pkg/kubelet/container/runtime.go

```diff
@@ -275,8 +278,11 @@ func (p Pods) FindPod(podFullName string, podUID types.UID) Pod {
 // When there are multiple containers with the same name, the first match will
 // be returned.
 func (p *Pod) FindContainerByName(containerName string) *Container {
+	glog.V(1).Infof("FindContainerByName here")
 	for _, c := range p.Containers {
+		glog.V(1).Infof("FindContainerByName: inspecting %v: %v", c.Name, containerName)
 		if c.Name == containerName {
+			glog.V(1).Infof("FindContainerByName: found %v", c.Name)
```

Comment:
Maybe rethink this log level? This is called often and might be to verbose to use in a reasonable default log level.

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/devel/logging.md